frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -258,10 +258,7 @@ function PreviousValueComparison({
       return comparisonDescStr;
     }
 
-    const descColor =
-      isNightMode || inTooltip
-        ? lighten(theme.fn.themeColor("text-medium"), 0.45)
-        : "text-light";
+    const descColor = "var(--mb-color-text-secondary)";
```

Comment:
![image](https://github.com/user-attachments/assets/836e51d5-50da-44c5-bba2-617d2b1ef2e0)

We change this color [according to the design.](https://metaboat.slack.com/archives/C063Q3F1HPF/p1723033749751899?thread_ts=1722869146.028559&cid=C063Q3F1HPF)